corehq/apps/user_importer/tasks.py

```diff
@@ -116,7 +118,12 @@ def _update_progress(value, start=0):
             update_progress=functools.partial(_update_progress, start=len(group_specs))
         )
         if TABLEAU_USER_SYNCING.enabled(domain):
-            import_tableau_users(domain, user_specs)
+            try:
+                import_tableau_users(domain, user_specs)
+            except TableauAPIError as e:
```

Comment:
Is there any way to signal this issue to the user as well?  Like can you stick a message in the upload status thingy without causing problems?

Reply:
Seems like you could append to `user_results["errors"]`

Reply:
That seems reasonable to me

Reply:
also, I don't think this needs to block this PR, necessarily